platforms/azure/giantnetes/variables.tf

```diff
@@ -181,6 +181,11 @@ variable "master_id" {
 
 ### Network ###
 
+variable "calico_mtu" {
+  description = "MTU used for Calico interfaces"
+  default     = "1500"
```

Comment:
Why wouldn't we switch to jumbo frames in all installations?

Reply:
Not all providers allow Jumbo frames, only AWS so far.

It could be also a hypothetical case where a customer workload wouldn´t be a perfect match (thinking on latency when you need to complete a full 9000 package). Not sure if it is a real scenario but I guess it is worth to be prepared.

Reply:
So how about setting jubmo for all AWS installations?

I personally believe that we should only create knobs when we need them and not other way around.

Reply:
I guess maybe we will probably change it in all aws installations but I thought It is better to extract this value to a variable and make it configurable in just one place for all environments instead of requiring going into the template.Then if it is the case, change the default value.

I had in mind also that when enabled CP with Jumbo frames will cohabit with not upgraded tenant clusters using common MTU. This should not be a problem but not 100% sure.

@giantswarm/sre

Reply:
Actually you are right. It's better than changing it it multiple places.

Reply:
Sorry. This is in single place :P 

It isn't a blocker for me. I think if this is a value used in a single place it's better to keep it like it was. I'll leave the decision to the guys working more with terraform.